### DISCUSSION TITLE
A way to search/filter tags on the left side panel

### DISCUSSION THREAD

**Discussion by frego** [standardnotes/forum#2509] [Suggestions]
It would be convenient to have a small search bar on the tag menu on the left to quickly filter tags.

**Comment by atmoio** [comment]
This has been brought up before, I do think it would be useful. How many tags do you have?

**Comment by frego** [comment]
I have 412 at the moment but I am adding new ones quite frequently.

**Comment by atmoio** [comment]
412 tags?? Wow! What's your workflow like that you have 412? tags I only have about 30, and most people I've talked with also have a low amount.

**Comment by frego** [comment]
Well most notes have multiples tags. Let's say I have a note on unlocking File Vault on OSX from command line. The note have the following tags: OSX / FileVault / Command line.

**Comment by atmoio** [comment]
Got it. Will keep this in mind.
